genai-perf/genai_perf/inputs/retrievers/synthetic_audio_generator.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+from typing import List, Literal, Dict
+import io
+import base64
+import soundfile as sf
+
+AudioFormat = Literal["wav", "mp3"]
+
+# MP3 supported sample rates in Hz
+MP3_SUPPORTED_SAMPLE_RATES = {
+    8000, 11025, 12000, 16000, 22050, 24000, 32000, 44100, 48000
+}
+
+class SyntheticAudioGenerator:
+    @staticmethod
+    def _sample_positive_normal(mean: float, stddev: float, min_value: float = 0.1) -> float:
+        """
+        Sample from a normal distribution ensuring positive values without distorting the distribution.
+        Uses rejection sampling to maintain the proper shape of the distribution.
+        
+        Args:
+            mean: Mean value for the normal distribution
+            stddev: Standard deviation for the normal distribution
+            min_value: Minimum acceptable value
+            
+        Returns:
+            A positive sample from the normal distribution
+        """
+        if mean < min_value:
+            raise ValueError(f"Mean value ({mean}) must be greater than min_value ({min_value})")
+        
+        while True:
+            sample = np.random.normal(mean, stddev)
+            if sample >= min_value:
+                return sample
+
+    @staticmethod
+    def _validate_sampling_rate(sampling_rate: int, output_format: AudioFormat) -> None:
+        """
+        Validate sampling rate for the given output format.
+        
+        Args:
+            sampling_rate: Sampling rate in Hz
+            output_format: Output audio format
+            
+        Raises:
+            ValueError: If sampling rate is not supported for the given format
+        """
+        if output_format.lower() == "mp3" and sampling_rate not in MP3_SUPPORTED_SAMPLE_RATES:
+            supported_rates = sorted(MP3_SUPPORTED_SAMPLE_RATES)
+            raise ValueError(
+                f"MP3 format only supports the following sample rates (in Hz): {supported_rates}. "
+                f"Got {sampling_rate} Hz. Please choose a supported rate from the list."
+            )
+
+    @staticmethod
+    def create_synthetic_audio(
+        audio_length_mean: float,
+        audio_length_stddev: float,
+        sampling_rates_khz: List[int],
+        bit_depths: List[int],
+        output_format: AudioFormat = "wav",
```

Comment:
Can we set the arg name to be `audio_format`? It'll be more consistent with other generators. Also, we already use [output format](https://github.com/triton-inference-server/perf_analyzer/blob/5f322733f4a942940670136d8fdb2f37f8096264/genai-perf/genai_perf/inputs/input_constants.py#L41) to refer to endpoints throughout the codebase.

```suggestion
        audio_format: AudioFormat = "wav",
```